modules/Bio/EnsEMBL/Compara/RunnableDB/PrepareMaster/UpdateGenomesFromMetadataFactory.pm

```diff
@@ -47,6 +51,13 @@ sub fetch_input {
 	my $release = $self->param_required('release');
 	my $division = $self->param('division');
 	my $metadata_script_options = "\$($meta_host details script) --release $release" . ($division ? " --division $division" : "");
+    my $rr_meta_db = $self->param('rr_meta_name');
+    my ($rr, $rr_meta_dba)     = _get_curr_rr_release($rr_meta_db, $meta_host);
+
+    my $meta_rr_script_options;
+    if ( $rr_meta_dba ) {
+        $meta_rr_script_options = "\$($meta_host details script) --dbname $rr_meta_db --release $rr --eg_first 1 --dump_path " . $self->param_required('work_dir');
+    }
```

Comment:
```suggestion
```
`$meta_rr_script_options` is redefined later on an never used before that (that I've been able to see).